app/src/main/java/sopt/org/millie/presentation/search/post/SearchPostScreen.kt

```diff
@@ -0,0 +1,61 @@
+package sopt.org.millie.presentation.search.post
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import sopt.org.millie.core.designsystem.theme.MillieTheme
+
+@Composable
+fun SearchPostScreen(
+    onClick: () -> Unit = {},
+    modifier: Modifier = Modifier,
```

Comment:
찬미야 요거 파라미터 순서만 변경해주구 머지합시다 !! 모디파이어는 선택인자 중에서 가장 앞에 위치 시켜야해요 !!
```suggestion
    modifier: Modifier = Modifier,
    onClick: () -> Unit = {},
```

Reply:
넵!!